Ryujinx.Horizon.Generators/CodeGenerator.cs

```diff
@@ -1,12 +1,14 @@
-﻿using System.Text;
+﻿using System;
+using System.Text;
 
 namespace Ryujinx.Horizon.Generators
 {
     class CodeGenerator
     {
-        private const string Indent = "    ";
+        private const int SingleIndentLength = 4;
```

Comment:
```suggestion
        private const int IndentLength = 4;
```
We can simplify the name here.